### PR TITLE
Fixes missing bulking when adding/removing users to/from CA groups

### PR DESCRIPTION
**Pull request by danieljoos**

### CODE REVIEW

**Review by arthur-babayan — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -155,48 +155,52 @@ Identification of users is done based on the email addresses.
...
-### [1.1.0]	
-- Added option `UseSecondaryEmails` to additionally use the user's	
-  secondary email addresses for the synchronization.	
-- Added configuration field `SsoCustomerId` to create user accounts that	
-  have Single Sign-On already activated.	
-- Added text filtering in the Active Directory groups drop-down menu.	
-  The filter is applied after typing at least 3 characters.	
-- Fixed encoding problem when creating or updating TeamViewer accounts.	
-- Log output now lists changes when updating a user.	
+## Changelog
```
Comment (MichaelBraitmaier) [inline README.md:158]: Why are all line changed? Would have expected only the diff of 1.3.1

**Review by MichaelBraitmaier — COMMENTED** [review]
All fine from my POV. The large count of changed lines in Readme is strange though

File: README.md (hunk 1 above)
Comment (danieljoos) [reply README.md:158]: wrong line-endings... got automatically corrected by vscode

**Pull request merged**